ui/app/bedmanagement/controllers/adtController.js

```diff
@@ -187,7 +187,7 @@ angular.module('bahmni.ipd')
                 spinner.forPromise(bedService.assignBed(bed.bedId, patientUuid, encounterUuid).then(function () {
                     bed.status = "OCCUPIED";
                     $scope.$emit("event:patientAssignedToBed", $rootScope.selectedBedInfo.bed);
-                    messagingService.showMessage('info', "Bed " + bed.bedNumber + " is assigned successfully");
+                    messagingService.showMessage("info", $translate.instant("BED") + " " + bed.bedNumber + " " + $translate.instant("IS_SUCCESSFULLY_ASSIGNED_MESSAGE"));
```

Comment:
Hardcoding "OCCUPIED"  can be moved to a config file.

Reply:
yes @buvaneswari-arun I will make the change and push it to the same PR

Reply:
@buvaneswari-arun after i gone through the code, i see that we are not using the bed.status field in the UI. Its just only for the verification purpose. 
But we can remove the hardcoded value with a constant but that would require changes in other place. I am thinking we can handle as part of the other card.